### PR TITLE
Evaluate PAYMENT_OFFSET at the start of each payment loop

### PR DESCRIPTION
**Pull request by utdrmac**
This is a partial fix for #71, which is a feature request for something else but a bug was discovered while investigating.
Correctly rebased off master this time :)

### CODE REVIEW

**Comment by jdsika** [comment]
PR Template with effort estimation missing + review from @amzid before merge

File: src/launch_common.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def add_argument_release_override(parser):
 def add_argument_payment_offset(parser):
     parser.add_argument("-O", "--payment_offset",
                         help="Number of blocks to wait after a cycle starts before starting payments. "
-                             "This can be useful because cycle beginnings may be bussy.",
-                        default=0, type=int)
+                             "This can be useful because cycle beginnings may be busy.",
+                        default=30, type=int)
```
Comment (jdsika) [inline src/launch_common.py:87]: I would leave the default value at 0 and only allow uint limited to the maximum number of blocks in a cycle

**Review by jdsika — CHANGES_REQUESTED** [review]

File: src/launch_common.py (hunk 1 above)
Comment (utdrmac) [reply src/launch_common.py:87]: Well, I was hoping we could "help the network" by not making the default be the first block of the cycle each time. I'd really like to make this a random number between 1 and 60 if the user sticks to the default.

**Comment by utdrmac** [comment]
Will re-open PR with template

**Pull request closed without merging**